task/kaniko/0.3/kaniko.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: kaniko
+  labels:
+    app.kubernetes.io/version: "0.3"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
```

```suggestion
    tekton.dev/pipelines.minVersion: "0.17.0"
```
since the Task is using Optional Workspaces

---

Good catch, thank you!